test/__init__.py

```diff
@@ -0,0 +1,3 @@
+import sys
+
+sys.path.append("../src")
```

Comment:
Nice! Löst das unser Module-not-found-Problem?

Reply:
das macht einfach die src/ inhalte in test/ sichtbar, obwohl die beiden parallel liegen. Unser Module not found problem war meiner vermutung nach dem umstand geschuldet, die sachen vom falschen ordner aus zu starten und hat mich in jüngerer zukunft nie mehr heimgesucht